foundation/aissemble-foundation-core-python/src/policy_manager/policy/json/policy_input.py

```diff
@@ -67,3 +70,23 @@ class PolicyInput(BaseModel):
     The rules for this policy.
     """
     rules: List[PolicyRuleInput] = []
+
+    """
+    This attribute is deprecated and should not be used. Target are now represented as
+    a List of Target's instead of a single Target attribute 'target'.
+    This attribute is replaced by `targets`.
```

Comment:
```suggestion
    This attribute is deprecated and should not be used. Targets are now represented as
    a List of Target objects instead of a single Target attribute 'target'.
    This attribute is replaced by `targets`.
```